Quantum-Control-Applications-QuAM/Superconducting/configuration/make_quam.py

```diff
@@ -22,6 +22,7 @@
 # Class containing tools to help handling units and conversions.
 u = unit(coerce_to_integer=True)
 
+# TODO: what is this for?
```

Comment:
It's so the IDE knows that "create_quam_superconducting" will return an instance of the same type as `quam_class`